src/javascript/app/base/footer.js

```diff
@@ -25,6 +28,10 @@ const Footer = (() => {
         $('#footer-regulatory .eu-only').setVisibility(should_show_warning);
     };
 
+    const showAgeRestrictionSign = (should_show_age_restriction_sign) => {
```

Comment:
this doesn't always show, it sometimes hides. it's better to update the name, something like `showHideAgeRestrictionSign` or `setVisibilityAgeRestrictionSign` etc